docs/tutorial.md

```diff
@@ -0,0 +1,372 @@
+            return UpdateControl.updateStatusSubResource(resource);
+        }
+```
+
+### createMemcachedDeployment
+
+Thie method simply creates the Deployment as specified below.
+
+```
+    private Deployment createMemcachedDeployment(Memcached m) {
+        return new DeploymentBuilder()
+            .withMetadata(
+                new ObjectMetaBuilder()
+                    .withName(m.getMetadata().getName())
+                    .withNamespace(m.getMetadata().getNamespace())
+                    .withOwnerReferences(
+                        new OwnerReferenceBuilder()
+                            .withApiVersion("v1alpha1")
+                            .withKind("Memcached")
+                            .withName(m.getMetadata().getName())
+                            .withUid(m.getMetadata().getUid())
+                            .build())
+                    .build())
+            .withSpec(
+                new DeploymentSpecBuilder()
+                    .withReplicas(m.getSpec().getSize())
+                    .withSelector(
+                        new LabelSelectorBuilder().withMatchLabels(labelsForMemcached(m)).build())
+                    .withTemplate(
+                        new PodTemplateSpecBuilder()
+                            .withMetadata(
+                                new ObjectMetaBuilder().withLabels(labelsForMemcached(m)).build())
+                            .withSpec(
+                                new PodSpecBuilder()
+                                    .withContainers(
+                                        new ContainerBuilder()
+                                            .withImage("memcached:1.4.36-alpine")
+                                            .withName("memcached")
+                                            .withCommand("memcached", "-m=64", "-o", "modern", "-v")
+                                            .withPorts(
+                                                new ContainerPortBuilder()
+                                                    .withContainerPort(11211)
+                                                    .withName("memcached")
+                                                    .build())
+                                            .build())
+                                    .build())
+                            .build())
+                    .build())
+            .build();
+    }
+```
+
+### deleteResource
+
+The delete Resource is an implemented method. The deltion part was take care by the Java Operator SDK that's why it does not have any code as such.
+
+## Run the Operator Locally
+
+### 1. Run locally outside the cluster
+
+The following steps will show how to deploy the operator on the Cluster. However, to run locally for development purposes and outside of a Cluster use the target `make install run`.
+
+First, create CR and CRD's within the cluster.
+
+`kubectl apply -f cr.yaml`
+
+
+`kubectl apply -f memcached-sample.yaml`
```

Comment:
Please explain how to create this file. See earlier comment about that section.

Reply:
Modified.